### PR TITLE
feat: add strict option for defaultConverter

### PR DESCRIPTION
**Pull request by srabouin**
formatDistanceToNow: `less than a minute ago`
formatDistanceToNowStrict: `0 seconds ago`

### CODE REVIEW

**Comment by srabouin** [comment]
I added the docs and the example, noticed the example was using the wrong version so I also updated that. Hope this helps!

File: src/converter/defaultConverter.ts [hunk 1]
```diff
@@ -14,7 +15,14 @@ export default (
     date = parseISO(date);
   }
 
-  const { includeSeconds, addSuffix = true } = converterOptions;
+  const { includeSeconds, addSuffix = true, useStrict = false } = converterOptions;
+
+  if(useStrict) {
```
Comment (MrDeerly) [inline src/converter/defaultConverter.ts:20]: Probably the reason why the pipeline (linting) fails
```suggestion
  if (useStrict) {
```

**Comment by MrDeerly** [comment]
Hey @srabouin! 

Your change was just released in v2.2.0. Thanks again for your contribution! :)

**Comment by srabouin** [comment]
Awesome, thanks for fixing the little things :)

**Pull request merged**